microk8s-resources/actions/metrics-server.yaml

```diff
@@ -99,51 +99,22 @@ spec:
       serviceAccountName: metrics-server
       containers:
       - name: metrics-server
-        image: k8s.gcr.io/metrics-server-$ARCH:v0.2.1
+        image: k8s.gcr.io/metrics-server-$ARCH:v0.3.3
```

Comment:
I do not see the v0.3.3 available for arm https://console.cloud.google.com/gcr/images/google-containers/GLOBAL/metrics-server-arm?gcrImageListsize=30 . Should we go for v0.3.2?

Reply:
I tested it on amd64, so didn't realize this. Let me change this to `v0.3.2` please